### PR TITLE
testing the html push merge branch

### PR DESCRIPTION
**Pull request by meriem0319**

### CODE REVIEW

**Comment by JamshedSadullaev** [comment]
Accepted

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (JamshedSadullaev) [inline index.html:1]: approved

**Comment by meriem0319** [comment]
request approval

**Review by JamshedSadullaev — APPROVED** [review]
I liked it

**Review by JamshedSadullaev — COMMENTED** [review]
something else

**Pull request merged**